pkg/utils/node/predicates.go

```diff
@@ -21,6 +21,11 @@ import (
 	"time"
 )
 
+const (
+	// TODO: Decide if this can be configured in the spec, and implement.
+	UnderutilizedPercentage = .1
```

Comment:
This is a fairly large architectural decision. I'd leave this at zero for now, and we can brainstorm the bigger picture.

Reply:
I'm good with leaving it at 0, but I think it's important to have it at some non-zero value as it'll be more realistic and can start capturing potential edge-cases with drain.

Reply:
I promise that this discussion is deeper than this :)